lib/compute/views.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * SPDX,License,Identifier: Apache,2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache,2.0 license or a
+ * compatible open source license.
+ */
+
+export class ViewsConfig {
+  private static getViewsConfig(name: string, regex: string): any {
+    const config = {
+      list: {
+        columns: [
+          'status',
+          'weather',
+          'jobName',
+          'lastSuccess',
+          'lastFailure',
+          'lastDuration',
+          'buildButton',
+          'favoriteColumn',
```

Comment:
What is this for in the view?

Reply:
Columns displayed. I have kept what was default.

Reply:
Got it.